templates/typescript_gapic/src/$version/$service_client.ts.njk

```diff
@@ -578,6 +580,8 @@ export class {{ service.name }}Client {
 {%- if method.serverStreaming and method.clientStreaming %}
 /**
 {{- util.printComments(method, service) }}
+ * @example <caption>include:samples/generated/{{api.naming.version}}/{{service.name.toSnakeCase()}}.{{method.name.toSnakeCase()}}.js</caption>
+ * region_tag:{{ api.hostName.split(".")[0] }}_{{ api.naming.version }}_generated_{{ service.name }}_{{ method.name }}_async
```

Comment:
For learning purpose, what's the region_tag?

Reply:
The region tags for the samples! ```START region_tag```

Reply:
Where is the `region_tag` definition. Try to understand from user perspective.

Reply:
Sync offline Thanks @fhinkel : `region_tag` is specific to our client libraries. It's handled here: https://github.com/googleapis/jsdoc-region-tag/blob/main/src/index.js

> We're ignoring <captoin> filename </caption> , we might as well remove it from the generator.  But for humans reading the docs, it might be nice to know the filename.